### DISCUSSION TITLE
Looking for option: SaveTable

### DISCUSSION THREAD

**Discussion by mabra** [gildas-lormeau/SingleFile#1465] [Ideas]
Hello!
Often I do not need the whole page, but a single table of it - which MAY not be a html table, but that is not that important.
Would be the greatest hit for me!

**Comment by mabra** [comment]
Oh, I must have been a dud .... that works with selections .....
(I used a different tool before).
Gratulation: Good tool !!
Leave this if anyone is looking for the same ....

> Hello! Often I do not need the whole page, but a single table of it - which MAY not be a html table, but that is not that important. Would be the greatest hit for me!

**Reply by gildas-lormeau** [reply]
Thank you for the feedback :)
